controller/handler/patient.js

```diff
@@ -81,14 +81,17 @@ function patientView (request, reply) {
                 patient: processPatient(currentPatient),
                 trial: currentTrial,
                 surveys: surveyInstances.map((surveyInstance) => {
-                    surveyInstance.startTime = moment(surveyInstance.startTime).format('MM-DD-YYYY');
-                    surveyInstance.endTime = moment(surveyInstance.endTime).format('MM-DD-YYYY');
+                    surveyInstance.startTime = moment(new Date(surveyInstance.startTime)
```

Comment:
@ChristianMurphy 
Some date fields in survey instance (join query) are not actually Date but instead strings.
Have a look at screenshot:
![screen shot 2016-02-25 at 10 53 40 am](https://cloud.githubusercontent.com/assets/5720497/13328838/1a8f7cb8-dbae-11e5-806d-c60e6533b5ba.png)
Usage of strings in momentjs will be deprecated in future. Thats why @mpenumal is using new Date inside momentjs. 
Do you have any recommendation over it  ?